fe/fe-core/src/main/java/com/starrocks/mysql/MysqlHandshakePacket.java

```diff
@@ -34,7 +36,11 @@
     private static final MysqlCapability CAPABILITY = MysqlCapability.DEFAULT_CAPABILITY;
     // status flags not supported in StarRocks
     private static final int STATUS_FLAGS = 0;
-    private static final String AUTH_PLUGIN_NAME = "mysql_native_password";
+    private static final String DEFAULT_AUTH_PLUGIN_NAME = "mysql_native_password";
+    private static final ImmutableMap<String, Boolean> supportedPlugins = new ImmutableMap.Builder<String, Boolean>()
+            .put(DEFAULT_AUTH_PLUGIN_NAME, true)
+            .put("mysql_clear_password", true)
```

Comment:
why not define a static variable for this?

Reply:
done